src/main/kotlin/com/autonomousapps/internal/JarExploder.kt

```diff
@@ -73,7 +73,7 @@ internal class JarExploder(
       analyzedClasses = analyzedClasses,
       ktFiles = ktFiles,
       androidLintRegistry = findAndroidLinter(artifact)
-    ).also { inMemoryCache.explodedJars(zip.name, it) }
+    ).also { inMemoryCache.explodedJars(artifact.file.absolutePath, it) }
```

Comment:
Thanks. I think you were right, this never worked. A bit of an error when I changed models for the v1 release.

Reply:
@jjohannes could you cherry pick this to a different pull request

Reply:
Sure. I removed the change from this PR and moved it here: https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/pull/699